README.md

```diff
@@ -148,6 +148,56 @@ ioBroker has the ability to include files written by adapters in its backups. To
 
 This path is relative to the path returned by `getAbsoluteDefaultDataDir()`. The placeholder `%INSTANCE%` is automatically replaced by the instance number of each adapter, for example `"dataFolder": "my-adapter.%INSTANCE%"`.
 
+## OAuth2 token refresher
+To keep the OAuth2 access token up to date, you can use the `TokenRefresher` class.
+
+```Typescript
+import { TokenRefresher } from './lib/TokenRefresher';
+
+export class YourAdapter extends Adapter {
+    private tokenWorker?: TokenRefresher;
+
+    // It is important to call this method in the `onReady` method of your adapter and not in the constructor.
+    async onReady(): Promise<void> {
+        this.tokenWorker = new TokenRefresher(this, 'yourService');
+        // Your other initialization code...
+        
+        // Then later in code, you can get the access token like this:
+        this.tokenWorker.getAccessToken()
+            .then(accessToken => {
```

Comment:
Why you do then/catch whn you alraedy have an async method? I think it is easier to understand with away and a try/catch :-)
Additionally, this runs parallel to the code below ... is that intended?